### DISCUSSION TITLE
Pioche sacrificielle

### DISCUSSION THREAD

**Discussion by Easygamer01s** [Crownicles/Crownicles#2411] [Nouvelles idées]
La proposition est de faire des objets avec une durée de vie limitée en fonction du niveau de rareté..
L'objectif est de créer un dilemme entre garder un atout traditionnel et un atout sacrificiel
Par exemple, la pioche 1 est un atout rare offrant + 500 or/jour, à 15 reprises 
La pioche 2 est un atout légendaire offrant +2000 or/jour, à 30 reprise.
La balle explosive augmente l'attaque de 200 pendant 30 combats
Une fois consommé, l'objet disparaît..

On peut le voir aussi comme une potion à plusieurs usages, mais ça ne crée pas le même dilemme chez le joueur..

**Comment by Easygamer01s** [comment]
Et petite note, il y avait avant un bug qui faisait qu'une potion n'était pas consommé sur l'île.. perso j'ai trouvé que c'était une feature super intéressante :D

**Comment by BastLast** [comment]
Etant donné le système d'item actuel, ca serait assez compliqué à coder
